airbyte-api/src/main/openapi/config.yaml

```diff
@@ -4706,6 +4759,31 @@ components:
           $ref: "#/components/schemas/ResourceId"
         workspaceId:
           $ref: "#/components/schemas/WorkspaceId"
+    AttemptId:
+      type: integer
+      format: int32
+    TemporalWorkflowId:
+      type: string
+    SetTemporalWorkflowInAttemptRequestBody:
+      type: object
+      required:
+        - jobId
+        - attemptId
+        - temporalWorkflowId
+      properties:
+        jobId:
+          $ref: "#/components/schemas/JobId"
+        attemptId:
+          $ref: "#/components/schemas/AttemptId"
+        temporalWorkflowId:
+          $ref: "#/components/schemas/TemporalWorkflowId"
+    InternalOperationResult:
+      type: object
+      required:
+        - done
+      properties:
+        done:
```

Comment:
From parent PR:
![image](https://user-images.githubusercontent.com/7798112/186285318-6e388a1c-f13a-4ac4-b55c-b7a7c3a9cf12.png)

Reply:
@xiaohansong fyi

Reply:
Ack

Reply:
Xiaohan fixed this